src/routes/estimation/[roomId]/+page.svelte

```diff
@@ -244,17 +274,22 @@
 	<EstimateGroupsList {estimateGroups} />
 {/if}
 
+{#if showFireworks}
+	<div class="launch-pad">
```

Comment:
I'm not sure this class name is particularly descriptive, although it is a very strong image!

Reply:
🚀